### PR TITLE
add ways of working

### PR DESCRIPTION
**Pull request by tom-webber**
- **Add ways of working (modified from AP's runbooks)**
- **remove url-check deploy stage**

### CODE REVIEW

File: source/documentation/ways-of-working/index.html.md.erb [hunk 1]
```diff
@@ -6,3 +6,206 @@ review_in: 3 months
...
+
+## Ceremonies
+
+### Schedule
+
+This team uses **Fortnightly Sprints** This is our timeframe for planning, executing, and reviewing work.
+
+The team has the following ceremonies:
+
+| Ceremony                                      | Occurence   | Day             | Time          |
```
Comment (MatMoore) [inline source/documentation/ways-of-working/index.html.md.erb:39]: I would leave out the day and time here - it's gonna change often, and the best place to check this is the team calendar

File: source/documentation/ways-of-working/index.html.md.erb [hunk 2]
```diff
@@ -6,3 +6,206 @@ review_in: 3 months
...
+Standup is run by either the PM, DM, or whoever wants to run it.
+
+Ad-hoc discussions may spin out from stand-up.
+These should take place after the meeting ends if time allows,
+or should be arranged for the next available time.
+
+### Show and Tell
+
+- Near the beginning of each sprint we have a thirty-minute show and tell showcase that provides an opportunity to share our progress from the previous sprint with the wider organisation.
+- [TODO: show and tell content]
```
Comment (MatMoore) [inline source/documentation/ways-of-working/index.html.md.erb:73]: what's this TODO for?

File: source/documentation/ways-of-working/index.html.md.erb [hunk 3]
```diff
@@ -6,3 +6,206 @@ review_in: 3 months
...
+
+#### Audience
+
+All of Data Platform and any wider audience
```
Comment (MatMoore) [inline source/documentation/ways-of-working/index.html.md.erb:77]:
```suggestion
The team, stakeholders, and anyone interested in the catalogue
```

File: source/documentation/ways-of-working/index.html.md.erb [hunk 4]
```diff
@@ -6,3 +6,206 @@ review_in: 3 months
...
+| --------------------------------------------- | ----------- | --------------- | ------------- |
+| [Stand Up](#stand-up)                         | Daily       | Monday - Friday | 10:45 - 11:00 |
+| [Show & Tell](#show-and-tell)                 | Fortnightly | Monday          | 15:00 - 15:30 |
+| [Roadmap Monthly Review](#roadmap-review)     | Fortnightly | Monday          | 15:30 - 16:15 |
+| [Refinement 1](#refinement)                   | Fortnightly | Monday          | 13:00 - 14:00 |
+| [Peer Review](#peer-review)                   | Fortnightly | Thursday        | 15:30 - 16:30 |
+| [Refinement 2](#refinement)                   | Fortnightly | Thursday        | 12:00 - 12:45 |
+| [Sprint Retrospective](#sprint-retrospective) | Fortnightly | Thursday        | 11:00 - 12:00 |
+| [Sprint Planning](#sprint-planning)           | Fortnightly | Thursday        | 13:00 - 14:00 |
+
```
Comment (jemnery) [inline source/documentation/ways-of-working/index.html.md.erb:49]: Is adding the ceremony meeting schedule creating a bit of a doc maintenance headache?

File: source/documentation/ways-of-working/index.html.md.erb [hunk 5]
```diff
@@ -6,3 +6,206 @@ review_in: 3 months
...
+### Show and Tell
+
+- Near the beginning of each sprint we have a thirty-minute show and tell showcase that provides an opportunity to share our progress from the previous sprint with the wider organisation.
+- [TODO: show and tell content]
+
+#### Audience
+
+All of Data Platform and any wider audience
+
+### Roadmap Review
```
Comment (MatMoore) [inline source/documentation/ways-of-working/index.html.md.erb:79]: maybe Sean can write a blurb for this section?

File: source/documentation/ways-of-working/index.html.md.erb [hunk 6]
```diff
@@ -6,3 +6,206 @@ review_in: 3 months
...
+
+## GitHub Practices
+
+- We use GitHub for managing code, documentation, and support.
+- We use GitHub Projects for tracking work and GitHub pull requests for reviewing and merging code.
+- When working in our repos, please adhere to GitHub housekeeping guidelines when creating pull requests,
+ensuring clear titles and descriptions, linking to issues when applicable,
+and requesting reviews upon readiness
+
+### GitHub Housekeeping
```
Comment (jemnery) [inline source/documentation/ways-of-working/index.html.md.erb:200]: Would this fit better to a 'contributing.md' doc?

File: source/documentation/ways-of-working/index.html.md.erb [hunk 7]
```diff
@@ -6,3 +6,206 @@ review_in: 3 months
...
+**Preparation:**
+
+Here's what can be prepared prior to engaging in a refinement session:
+
+- **Backlog overview:**
+Review the backlog to get a sense of what's coming up, and what's already been done.
+Read the tickets likely to come up for discussion, so you know what questions you want to ask.
+- **Backlog item review:**
+Ensure any tickets are well fleshed out and ready for discussion.
+This should include a detailed description of the story, as well as acceptance criteria.
```
Comment (MatMoore) [inline source/documentation/ways-of-working/index.html.md.erb:104]:
```suggestion
[This should include a detailed description of the story, as well as acceptance criteria.](https://www.gov.uk/service-manual/agile-delivery/writing-user-stories)
```

File: source/documentation/ways-of-working/index.html.md.erb [hunk 8]
```diff
@@ -6,3 +6,206 @@ review_in: 3 months
...
+If your tickets rely on other work already being done,
+know which tickets are required and list them in the body of the issue.
+This helps the team establish sequencing of the tickets during refinement.
+- **Prepare to Communicate!:**
+Be prepared to articulate why you think a ticket should be done, how big you think it is,
+be receptive to teammates, and collaborate to achieve a shared understanding of the tasks at hand.
+Be open to suggestions on how your ticket could be improved for someone picking it up.
+
+**Process:**
+
```
Comment (MatMoore) [inline source/documentation/ways-of-working/index.html.md.erb:115]: I like these guidelines - will be helpful if Sean is away and somebody else needs to run this meeting. Might be useful to copy this over into the calendar invite as well (or link to this)

File: source/documentation/ways-of-working/index.html.md.erb [hunk 9]
```diff
@@ -6,3 +6,206 @@ review_in: 3 months
...
+- **Scored stories:**
+Each story gets a score so we can try to estimate how many stories we should take on based on our prior rate of work.
+- **Approachable tickets:**
+All tickets that go through refinement should have all the info required for any member of the team to pick up.
+
+#### Audience
+
+Data Catalogue Team
+
+### Peer Review
```
Comment (MatMoore) [inline source/documentation/ways-of-working/index.html.md.erb:141]: Are we still doing this?

File: source/documentation/ways-of-working/index.html.md.erb [hunk 10]
```diff
@@ -6,3 +6,206 @@ review_in: 3 months
...
+
+- In the middle of each sprint we have a thirty-minute peer review session
+that provides an opportunity to receive feedback on the current state
+and future plans for the service.
+- [TODO: peer review content]
+
+#### Audience
+
+All of Data Platform and wider audience
```
Comment (MatMoore) [inline source/documentation/ways-of-working/index.html.md.erb:150]:
```suggestion
All of the team and wider audience
```

File: source/documentation/ways-of-working/index.html.md.erb [hunk 11]
```diff
@@ -6,3 +6,206 @@ review_in: 3 months
...
+**Output:**
+
+- Ensure everyone has stories that they feel they can work on
+- Ensure everyone has a shared understanding of the tasks to deliver this sprint
+
+#### Audience
+
+Data Catalogue Team
+
+## GitHub Practices
```
Comment (MatMoore) [inline source/documentation/ways-of-working/index.html.md.erb:192]: What do you think about pulling this out into it's own page, and maybe having a discussion to review it?

There was a discussion recently on X-Gov slack about this kind of thing, and a team from HMCTS shared a nice example https://hmcts.github.io/cloud-native-platform/standards/pull-requests.html

**Review by MatMoore — DISMISSED** [review]

File: source/documentation/ways-of-working/index.html.md.erb (hunk 4 above)
Comment (tom-webber) [reply source/documentation/ways-of-working/index.html.md.erb:49]: Potentially, yes, but also I think it's good to actually see it and state what it is. I didn't realise the precise order of operations until I put it all down here. Might be worth the headache.

File: source/documentation/ways-of-working/index.html.md.erb (hunk 2 above)
Comment (tom-webber) [reply source/documentation/ways-of-working/index.html.md.erb:73]: I wasn't sure if there was a consistent 'content' / order of operations for the Show and Tell. If there is, someone should put it here

File: source/documentation/ways-of-working/index.html.md.erb (hunk 1 above)
Comment (tom-webber) [reply source/documentation/ways-of-working/index.html.md.erb:39]: See [other comment](https://github.com/ministryofjustice/data-catalogue-runbooks/pull/11#discussion_r1542639214) - I think it's useful to at least have the order and details here while we're refining our approach, and we can remove the dates/times in the future (maybe at the next review).

File: source/documentation/ways-of-working/index.html.md.erb (hunk 9 above)
Comment (tom-webber) [reply source/documentation/ways-of-working/index.html.md.erb:141]: Ah, I've put down the Data Platform's peer review, but I thought it was a Data Catalogue peer review. Not sure if that actually goes ahead though

File: source/documentation/ways-of-working/index.html.md.erb (hunk 11 above)
Comment (tom-webber) [reply source/documentation/ways-of-working/index.html.md.erb:192]: [You and](https://github.com/ministryofjustice/data-catalogue-runbooks/pull/11#discussion_r1542640359) @jemnery [both agree](https://github.com/ministryofjustice/data-catalogue-runbooks/pull/11#discussion_r1542640359). Looks to be worth pulling out into a contributing doc

File: source/documentation/ways-of-working/index.html.md.erb (hunk 2 above)
Comment (MatMoore) [reply source/documentation/ways-of-working/index.html.md.erb:73]: Ah I don't know... @seanprivett?

File: source/documentation/ways-of-working/index.html.md.erb (hunk 1 above)
Comment (MatMoore) [reply source/documentation/ways-of-working/index.html.md.erb:39]: Sounds like a good plan 👍🏻

File: source/documentation/ways-of-working/index.html.md.erb (hunk 6 above)
Comment (tom-webber) [reply source/documentation/ways-of-working/index.html.md.erb]: Page created and content moved

File: source/documentation/ways-of-working/contributing.html.md.erb [hunk 12]
```diff
@@ -0,0 +1,25 @@
+---
+owner_slack: "#data-catalogue"
+title: Ways of working
```
Comment (MatMoore) [inline source/documentation/ways-of-working/contributing.html.md.erb:3]:
```suggestion
title: Contributing
```

**Pull request merged**